### DISCUSSION TITLE
Function for converting text file line endings between Windows/DOS  (CRLF), Mac (CR), and Linux/Unix (LF)

### DISCUSSION THREAD

**Discussion by oldman20** [gsantner/markor#2227] [Ideas]
I want to use Markor for create, edit scripts for my phone. And Unfortunately, scripts created from Markor have errors when running on Android, because by default they are in CRLF/CR format, while they should be in LF format.

**Comment by gsantner** [comment]
The Android textview, which is used here, works with/does properly support \n only as far I know.
